.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
.vscode and other files generated by editor should be in gitignore

Reply:
Fix it, but dont fix tslint config, and other lint files